src/main/java/wolox/training/models/User.java

```diff
@@ -64,6 +67,7 @@ public LocalDate getBirthDate() {
     }
 
     public void setBirthDate(LocalDate birthDate) {
+        Preconditions.checkNotNull(birthDate, "birthDate must be not null");
```

Comment:
Add the following validations:

- That the field is not empty
- Validate if the birthday is earlier than the current date